.prettierrc.yaml

```diff
@@ -7,6 +7,6 @@ bracketSpacing: true
 arrowParens: always
 htmlWhitespaceSensitivity: ignore
 overrides:
-    - files: '{tools,packages}/*/src/spectrum-*.css'
+    - files: '{tools,packages}/*/src/*.css'
```

Comment:
All CSS assets benefit from this expanded line-length in order to prevent var function wraps that could block proper downstream minification efforts.